sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/LuceneStandardAnalyzer.java

```diff
@@ -59,9 +56,8 @@ public Integer getMaxTokenLength() {
     }
 
     /**
-     * Set the maxTokenLength property: The maximum token length. Default is
-     * 255. Tokens longer than the maximum length are split. The maximum token
-     * length that can be used is 300 characters.
+     * Set the maxTokenLength property: The maximum token length. Default is 255. Tokens longer than the maximum length
```

Comment:
is this line too long?

Reply:
Nope 😃 It's ~115 characters and we cut at 120